### DISCUSSION TITLE
[Suggestion] Can you please put the compiled idlparser.jar to maven repository? Thanks. [13743]

### DISCUSSION THREAD

**Discussion by liberty724048** [eProsima/IDL-Parser#90] [Ideas]

**Comment by JLBuenoLopez** [comment]
Thanks for the suggestion, @liberty724048 

I am labeling the ticket as an enhancement. I am afraid that, for the moment, IDL Parser is not going to be released into Maven repository. I am moving the ticket to the Ideas discussion forum where enhancements and feature requests are discussed.
